gravitee-apim-rest-api/gravitee-apim-rest-api-repository/src/main/java/io/gravitee/rest/api/repository/healthcheck/ManagementRepositoryProbe.java

```diff
@@ -28,6 +29,7 @@
  */
 public class ManagementRepositoryProbe implements Probe {
 
+    @Setter
     @Autowired
     private EventRepository eventRepository;
```

Comment:
I suppose you did that for testing but I think you can migrate to autowire constructor here without issue. wdyt?

Reply:
I did a try with CPUProbe and MemoryProbe and it didn't work. It seems that constructors with args aren't well supported when used in conjunction with spring.factories 😕